AsyncDisplayKit/Details/ASChangeSetDataController.mm

```diff
@@ -58,6 +56,9 @@ - (void)endUpdatesAnimated:(BOOL)animated completion:(void (^)(BOOL))completion
       _changeSet = nil;
       return;
     }
+
+    [self invalidateDataSourceItemCounts];
```

Comment:
I guess the test is effective at ensuring it stays this way… so maybe it's not necessary.